src/payment/strategies/amazon-pay/amazon-pay-payment-strategy.spec.ts

```diff
@@ -9,18 +9,20 @@ import { BillingAddressActionType } from '../../../billing/billing-address-actio
 import { getBillingAddress, getBillingAddressState } from '../../../billing/billing-addresses.mock';
 import { createCheckoutStore, CheckoutRequestSender, CheckoutStore, CheckoutStoreState, CheckoutValidator } from '../../../checkout';
 import { getCheckoutStoreState } from '../../../checkout/checkouts.mock';
-import { InvalidArgumentError, MissingDataError, RequestError } from '../../../common/error/errors';
+import {InvalidArgumentError, MissingDataError, NotInitializedError, RequestError} from '../../../common/error/errors';
```

Comment:
Split this into separate lines.
```suggestion
import {
    InvalidArgumentError,
    MissingDataError,
    NotInitializedError,
    RequestError
} from '../../../common/error/errors';
```